pallets/msa/src/mock.rs

```diff
@@ -107,8 +107,10 @@ impl pallet_msa::Config for Test {
 	type MaxProviderNameSize = MaxProviderNameSize;
 	type SchemaValidator = Schemas;
 	type MortalityWindowSize = ConstU32<100>;
-	type MaxSignaturesPerBucket = ConstU32<10>;
+	type MaxSignaturesPerBucket = ConstU32<4000>;
```

Comment:
This had to be increased - because of the new limit on signatures, the benchmark test was failing.